Tests/Library/Client.tests.ts

```diff
@@ -258,6 +258,8 @@ describe("Library/Client", () => {
                 // validate
                 var args = trackStub.args;
                 var tags = args[0][1];
+
+                assert.equal(tags["ai.operation.name"], "method /search");
```

Comment:
It's just a sample value - See line 180. I changed it to "GET" and updated the unit test.